### PR TITLE
Fix shimmer in dark mode and also add it on profile feed

### LINKED ISSUE #1082: Shimmer on dark mode
### Description

The shimmer effect is perfectly visible on light mode. But, on dark mode it is not visible. I want to fix that.
Screenshot 2023-08-02 135922

### Steps to reproduce

- Change theme to dark mode
- Reload the page
- Shimmer is not visible.

### Screenshots

_No response_

### Desktop (please complete the following information)

_No response_

### Smartphone (please complete the following information)

_No response_

### Additional information

_No response_

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by Palakkgoyal**
This PR closes #1082 
I have fixed the shimmer effect on darkmode and also added in on the profile feed.

### CODE REVIEW

File: src/reusableComponents/PostSkeleton/index.jsx [hunk 1]
```diff
@@ -6,33 +6,58 @@ const PostSkeleton = () => {
...
         sx={{ fontSize: "1rem" }}
-        className="skeleton-user-caption"
+        className="skeleton-user-caption shimmer_bg"
       />
       <div className="post-skeleton-nav">
-        <Skeleton variant="rectangular" width={50} height={35} />
-        <Skeleton variant="rectangular" width={50} height={35} />
-        <Skeleton variant="rectangular" width={50} height={35} />
-        <Skeleton variant="rectangular" width={50} height={35} />
+        <Skeleton
```
Comment (narayan954) [inline src/reusableComponents/PostSkeleton/index.jsx:35]: again repetition of lots of code

**Review by narayan954 — CHANGES_REQUESTED** [review]
Hi @Palakkgoyal there seems to be repetition of lots of code, if it seems feasible,can you try something like 
``` js
function FeedSkeleton() {
  const numSkeletons = 4;

  return (
    <>
      {Array.from({ length: numSkeletons }, (_, index) => (
        <Skeleton
          key={index}
          variant="rectangular"
          style={{
            width: "100%",
            height: "100%",
            aspectRatio: "1",
          }}
          className="shimmer_bg"
        />
      ))}
    </>
  );
}
```

**Comment by Palakkgoyal** [comment]
Yes, sure. I will fix that.

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Pull request merged**